src/jit/gentree.h

```diff
@@ -1628,6 +1628,9 @@ struct GenTree
 #ifdef FEATURE_SIMD
             case GT_SIMD:
 #endif // !FEATURE_SIMD
+#ifdef _TARGET_ARM_
+            case GT_PUTARG_REG:
+#endif
                 return true;
```

Comment:
`GT_PUTARG_REG` is defined as `GTK_UNOP`, so I don't see why this is necessary. (i.e., it's not a binary node in the first place)